amplify/backend/auth/bettergivingwebapp59590dc1/parameters.json

```diff
@@ -0,0 +1,31 @@
+{
+  "authSelections": "identityPoolAndUserPool",
+  "resourceName": "bettergivingwebapp59590dc1",
+  "serviceType": "imported",
+  "region": "us-east-1",
+  "usernameAttributes": [
+    "email"
+  ],
+  "authProvidersUserPool": [
+    "Google"
+  ],
+  "requiredAttributes": [
+    "given_name",
+    "family_name",
+    "email"
+  ],
+  "passwordPolicyMinLength": 8,
+  "passwordPolicyCharacters": [
+    "Requires Lowercase",
+    "Requires Uppercase",
+    "Requires Numbers",
+    "Requires Symbols"
+  ],
+  "mfaConfiguration": "OPTIONAL",
+  "autoVerifiedAttributes": [
+    "email"
+  ],
+  "mfaTypes": [
+    "TOTP"
+  ]
+}
```

Comment:
nit: please add missing newline characters in to all PR files that are missing it